apps/prairielearn/src/ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.sql

```diff
@@ -0,0 +1,11 @@
+-- BLOCK select_generation_sequence_by_course
+SELECT
+  j.*
+FROM
+  jobs AS j
+WHERE
+  j.course_id IS NOT DISTINCT FROM $course_id
+  AND j.type = 'ai_question_generate'
```

Comment:
Should we also be handling regeneration jobs?

Reply:
added in 3c8bafd1470728a8c79b488213de1dfae0aed79e.